src/dlgConnectionProfiles.cpp

```diff
@@ -120,16 +121,16 @@ dlgConnectionProfiles::dlgConnectionProfiles(QWidget* parent)
 
     auto pWelcome_document = new QTextDocument(this);
 
-    auto copyProfile = new QAction(tr("Copy"), this);
-    copyProfile->setObjectName(qsl("copyProfile"));
+    mpCopyProfile = new QAction(tr("Copy"), this);
+    mpCopyProfile->setObjectName(qsl("mpCopyProfile"));
```

Comment:
Renaming the control will break things if people are styling this dialogue I think...

Reply:
Thanks, good point, reverted.

Reply:
Not yet it hasn't! 😲 
```suggestion
    mpCopyProfile->setObjectName(qsl("copyProfile"));
```